src/lib/features.js

```diff
@@ -361,6 +361,21 @@ var featureInfos =
             return available;
         },
     },
+    FORMS:
+    {
+        description:
+        'Existence of the global object document having the property forms, which have string ' +
+        'representation [object HTMLCollection]".',
+        check:
+        function ()
+        {
+            var available = typeof document.forms === 'object' &&
+            document.forms + '' === '[object HTMLCollection]';
```

Comment:
```suggestion
            var available =
            typeof document === 'object' && document.forms + '' === '[object HTMLCollection]';
```

`document` is not defined in Node.js.